service/matching/db.go

```diff
@@ -201,6 +232,9 @@ func (db *taskQueueDB) CreateTasks(
 			},
 			Tasks: tasks,
 		})
+
+	atomic.StoreInt64(&db.maxReadLevel, maxReadLevel)
```

Comment:
I thought we were putting the comment back here?

Reply:
think my github had glitched since I thought Shahab had asked me to put it back but couldn't find it again, so I removed it; shall add it back, sorry for the inconvenience.